src/Orikivo/Services/TemplateBuilder.cs

```diff
@@ -1,512 +1,378 @@
-                                    }
-                                    else {
-
-                                        primaryGameName += name + " ";
-                                        primaryLength += name.Length;
-                                    }
-                                }
-                                else
-                                {
-                                    if (secondaryLength + name.Length <= secondActivityLength)
-                                    {
-                                        secondaryGameName += name + " ";
-                                        secondaryLength += name.Length;
-                                }
-                                    else
-                                    {
-                                        secondaryGameName += "...";
-                                        break;
-                                    }
-
-                                    
-                                }
+                                primaryGameName += name + Strings.Space(1);
+                                primaryLength += name.Length;
                             }
+                        }
+                        else
+                        {
+                            if (secondaryLength + name.Length <= secondActivityLength)
+                            {
+                                secondaryGameName += name + Strings.Space(1);
+                                secondaryLength += name.Length;
+                            }
+                            else
+                            {
+                                secondaryGameName += "...";
+                                break;
+                            }
+                        }
                     }
-
-                        
-                    if (secondaryGameName.StartsWith(' '))
-                    {
-                        secondaryGameName = secondaryGameName.Remove(0, 1);
-                    }
-                    //TextConfiguration.PixelateText(primaryGameName, 40, baseSpacer, "s", templateBuilder);
-                    //TextConfiguration.PixelateText(secondaryGameName, 40, baseSpacer + 5, "s", templateBuilder)
                 }
 
 
-
-
-                //Bitmap lstr = PixelEngine.RenderString(levelString, options);
-
-                //TextConfiguration.PixelateText(levelString, levelPoint.X, levelPoint.Y, "l", templateBuilder);
-                //TextConfiguration.PixelateText(currencyString, currencyPoint.X, currencyPoint.Y, "l", templateBuilder);
-
-                var un = PixelEngine.RenderString(userName, options);
-                graphics.DrawImage(un, new Point(40, 6));
-
-                int levelCurrencyY = 6 + un.Height + 2;
-
-                if (!string.IsNullOrWhiteSpace(secondaryGameName))
+                if (char.IsWhiteSpace(secondaryGameName.FirstOrDefault()))
```

Comment:
I feel like this can also be reverted back to `.StartsWith(' ')`, as this makes the code a bit more confusing to understand what is happening in this step.